### PR TITLE
fix(ButtonLink): Fix paddings

### PR DESCRIPTION
**Pull request by aweell**
The current buttonLink `padding` value is not following the specs.

https://www.figma.com/file/koROdh3HpEPG2O8jG52Emh/Buttons-Component-Specs?node-id=0%3A1

Expected: `padding: 6px 12px;`

### CODE REVIEW

**Comment by pladaria** [comment]
You need to generate type definitions and commit them. From the repo root run `yarn build` then commit and push the updated files.

I guess you also need to regenerate some screenshot tests.

File: src/button.tsx [hunk 1]
```diff
@@ -436,12 +436,13 @@ const Button: React.FC<
...
 const useButtonLinkStyles = createUseStyles((theme) => {
-    const padding = 6;
+    const paddingY = 6;
+    const paddingX = 12;
     return {
         link: {
             display: 'inline-block',
             width: 'auto',
-            padding,
+            padding: `${paddingY}px ${paddingX}px`,
```
Comment (yceballost) [inline src/button.tsx:445]: wrong alignment in callouts and cards
https://ibit.ly/HwBP
https://ibit.ly/YsS-

And hover in this case is quite "ugly"
https://ibit.ly/JDv_

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/button.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ import type {DataAttributes, RendersElement, RendersNullableElement, TrackingEve
 import type {Location} from 'history';
 import type {Theme} from './theme';
 
-export const BUTTON_MIN_WIDTH = 136;
+export const BUTTON_MIN_WIDTH = 104;
```
Comment (yceballost) [inline src/button.tsx:19]: Specs say 136
image

now the buttonPrimary has weird min-width
https://ibit.ly/2oW_
image

(before)
https://ibit.ly/vY8v
image

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/button.tsx (hunk 1 above)
Comment (aweell) [reply src/button.tsx:445]: Fixed. This problem was related to the buttongrouop component

File: src/button.tsx (hunk 2 above)
Comment (aweell) [reply src/button.tsx:19]: My error reading the specs, fixed primary min-width

**Review by yceballost — APPROVED** [review]
lgtm!

**Review by pladaria — APPROVED** [review]

**Pull request merged**